bosco-override-setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Populate the bosco override dir from a Git repo
+# Expected Git repo layout:
+#     RESOURCE_NAME_1/
+#         bosco_override/
+#         ...
+#     RESOURCE_NAME_2/
+#         bosco_override/
+#         ...
+#     ...
+
+function errexit {
+    echo "$1"
+    exit 1
+}
+
+[[ $@ -ne 2 ]] || errexit "Usage: bosco-override-setup.sh <GIT ENDPOINT> <RESOURCE NAME>"
+
+GIT_ENDPOINT=$1
+RESOURCE_NAME=$2
+
+REPO_DIR=$(mktemp -d)
+OVERRIDE_DIR=/etc/condor-ce/bosco_override/
+
+git clone --depth=1 $GIT_ENDPOINT $REPO_DIR
+
+RESOURCE_DIR="$REPO_DIR/$RESOURCE_NAME/"
+[ -d $RESOURCE_DIR ] || errexit "Could not find $RESOURCE_NAME/ under $GIT_ENDPOINT"
```

Comment:
```suggestion
[[ -d $RESOURCE_DIR ]] || errexit "Could not find $RESOURCE_NAME/ under $GIT_ENDPOINT"
```

Are there any restrictions on `$RESOURCE_NAME`?  (In particular, can it contain whitespace or shell glob characters?)  Anyway, `$RESOURCE_DIR` is used unquoted here, and the single-bracket test does not handle this in a safe way -- but it is safe to use a variable unquoted within a double-bracket test.